src/get-started/test-service.md

```diff
@@ -287,14 +291,14 @@ because it may be needed to sign other transactions authorized by the wallet
 owner.
 
 Inside, all wrapper methods call the `inner` API instance; for example,
-`get_wallet` is implemented as
+`get_wallet` is implemented as:
 
 ```rust
-fn get_wallet(&self, pubkey: &PublicKey) -> Wallet {
-    self.inner.get(
-        ApiKind::Service("cryptocurrency"),
-        &format!("v1/wallet/{}", pubkey.to_string()),
-    )
+fn get_wallet(&self, pub_key: &PublicKey) -> Wallet {
```

Comment:
Nit: It seems that `get_wallet` and `assert_no_wallet` [take `PublicKey` by value in the example](https://github.com/exonum/exonum/blob/ba6834334b43e6ad197b1523f093f2d98d69716b/examples/cryptocurrency/tests/api.rs#L259), which makes sense since it's a `Copy` type.